### PR TITLE
Add Demonshear

### LINKED ISSUE #35: Missing Demonshear
https://classic.wowhead.com/item=13348/demonshear

I'd create a pull request myself, but I have no idea about the proc rate.

### PR DESCRIPTION
**Pull request by veluria**
Fixes #35. Not sure if this is right, so please tell me what to correct. :)

### CODE REVIEW

**Comment by timhul** [comment]
> Fixes #35. Not sure if this is right, so please tell me what to correct. :)

It looks good! Note that the attribute "dmg_over_duration" is currently only supported for weapons proccing instant Fireballs (like Sulfuras) so it won't have any effect here. The DoT thus does not deal any damage, but this is good enough for now since it is a trash debuff that swiftly gets removed in a raid setting if it is applied. 

Supporting a generic proc dealing instant damage and applying a DoT proc requires a bit of rework and is something we'll have to do separately.

File: Equipment/EquipmentDb/Weapons/2hSwords.xml [hunk 1]
```diff
@@ -27,6 +27,20 @@
 		</source>
 	</melee_weapon>
 
+	<melee_weapon id="13348" phase="1">
+		<info name="Demonshear" type="TWOHAND_SWORD" slot="2H" unique="no" req_lvl="58" item_lvl="63" quality="RARE" boe="no" icon="Inv_sword_14.png"/>
+		<dmg_range min="163" max="246" speed="3.8"/>
+		<proc>
+			<spell name="SHADOW_ATTACK" min="150" max="150" instant="yes" rate="0.06" internal_cd="0" spell_dmg_coefficient="0.0" duration="6" dmg_over_duration="120"/>
```
Comment (timhul) [inline Equipment/EquipmentDb/Weapons/2hSwords.xml]: Please remove the 'duration' and 'dmg_over_duration' attributes as they have no effect right now. We will add the support as a separate commit.

**Comment by veluria** [comment]
Done. Thanks for your guidance.

**Comment by timhul** [comment]
Please rebase and squash the commits into a single commit and I will merge it.

**Comment by veluria** [comment]
> Please rebase and squash the commits into a single commit and I will merge it.

Done! Hope it's alright now.

**Comment by timhul** [comment]
> > Please rebase and squash the commits into a single commit and I will merge it.
> 
> Done! Hope it's alright now.

Did you fetch before rebase? There is still a merge conflict that I tried to resolve using the github web but it seems to have created a merge commit instead, which is not what I intended.

Could you make sure you rebase latest master?

**Pull request closed without merging**